pkg/http/core/manifest_test.go

```diff
@@ -11,6 +11,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+const payloadOkResponse = `{
```

Comment:
I would call this payloadManifestCoordinates to be consistent with the rest of the code and define it in payload_test.go file as a constant.

Reply:
Thanks - I made the change but made a mess of it in doing so.

I had merged the versioning branch into my local to do a build and deployed into sandbox environment to test.  I forgot about this merge when I made the suggested changes, so I ended up pushing the versioning changes into this pull request.  It took a few tries, but I've reverted that and got just the changes for manifest coordinates into this PR